boards/px4/fmu-v5/init/rc.board_sensors

```diff
@@ -5,8 +5,14 @@
 
 board_adc start
 
-# Internal SPI bus ICM-20602
-icm20602 -s -R 2 -q start
+if ver hwtypecmp V552
```

Comment:
if ver hwtypecmp V552 V562 

This will cover the 


2 | 5 | CUAV V5+ ICM42688P
-- | -- | --
2 | 6 | CUAV V5Nano ICM42688P